CHANGELOG.md

```diff
@@ -13,7 +13,7 @@ You can find and compare releases at the [GitHub release page](https://github.co
 
 - Add flag `--json` to `print-schema` to output JSON instead of GraphQL SDL https://github.com/nuwave/lighthouse/pull/1268
 - Add TTL option for subscriptions storage https://github.com/nuwave/lighthouse/pull/1284
-
+- `StorageManager` will remove subscribers from topics now when they are deleted.
 ### Changed
 
```

Comment:
```suggestion

### Changed

- When a subscription subscriber is deleted the subscriber reference will also be removed from the topic https://github.com/nuwave/lighthouse/pull/1288
```

It should be under `Changed`, and what about the wording, that might better describe what is happening?